src/InfoBoxes/Content/Factory.cpp

```diff
@@ -201,7 +201,7 @@ static constexpr MetaData meta_data[] = {
   {
     N_("Last thermal gain"),
     N_("TL Gain"),
-    N_("Total altitude gain/loss in the last thermal."),
+    N_("Total altitude gain/loss in the last thermal. The number in smaller font reflects the overall climb rate for the last thermal."),
```

Comment:
Can we "de-bloat" the wording here? I don't like the pharse "The number in smaller font reflects ...". Maybe just "Bottom: overall climb rate in the last thermal."

Reply:
Is "overall climb rate" really the right wording? "Average climb rate"?